### DISCUSSION TITLE
SB Obsrv - Release 5.0

### DISCUSSION THREAD

**Discussion by rhwarrier** [Sunbird-Obsrv/Community#15] [Documentation]
The details of the items planned for Release 5.0 of SB Obsrv have been published on the Obsrv microsite.
The 'Product Roadmap' page has the list of the items : https://obsrv.sunbird.org/learn/product-roadmap
The team will use this thread to call out any updates/ changes in plan to release 5.0

**Comment by rhwarrier** [comment]
SB Obsrv release 5.0 is now available : https://obsrv.sunbird.org/use/release-notes/release-v-5.0.0-latest

**Comment by anunagaraj** [comment]
@rhwarrier 
All the tickets added in the Microsite for Release 5.0.0 is tagged to R5.1.0 Sprint. Please update the Release notes of 5.0.0



@chitranshu-keshav @ShailandraOjha @sharathkashyap @luckynvdu @reshmi-nair

**Reply by ShailandraOjha** [reply]
Hi @rhwarrier can u please update on this request. We will need to ensure any changes in scope should be updated accordingly in the microsite as well

**Comment by anunagaraj** [comment]
@rhwarrier Please update the Microsite as we are unable to see the changes done.

CC @chitranshu-keshav @ShailandraOjha @sharathkashyap @luckynvdu  @anandp504

**Reply by anandp504** [reply]
@anunagaraj The release notes for 5.0.0 was updated quite a while ago. Please find the release notes here https://obsrv.sunbird.org/use/release-notes/release-v-5.0.0-latest.

**Comment by anunagaraj** [comment]
@anandp504 Please find the status of the below tickets mentioned in the release notes ,
SB-30518 --- Is not tagged to any sprint 
OB-3 --- Is tagged to R5.0 but the status is still "In Development"
OB-1 ---Is tagged to R5.0 but the status is still "Selected for Contribution"
Kindly update the tickets with correct sprint.

CC @ShailandraOjha

**Reply by anunagaraj** [reply]
@anandp504 Kindly update the tickets for the R5.0.0 in the release notes.

@ShailandraOjha @chitranshu-keshav
